tests/has/const.go

```diff
@@ -20,4 +20,10 @@ const (
 
 	// See more info: https://github.com/redhat-appstudio/application-service#creating-a-github-secret-for-has
 	ApplicationServiceGHTokenSecrName string = "has-github-token" // #nosec
+
+	//Name for the GitOps Deployment resource
```

Comment:
//Name for the GitOps Deployment resource => 	// Name for the GitOps Deployment resource